### PR TITLE
final live tail fixes

### PR DESCRIPTION
**Pull request by kavithacm**
Signed-off-by: Kavitha Conjeevaram Mohan <mohakavi@amazon.com>

### Description
Final UI changes and bug fixes for live tail in event analytics includes

- live tail stops when the page isn't in focus (includes event tab change and browser tab change too)
- disabling live tail in visualization tab
- remove histogram and current hits when live tail is on
- button icon update
- include start time timestamp to indicate when live tail is switched on
- cypress to test automatic off when tab is changed and on/off button functionality 

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by davidcui1225 — APPROVED** [review]
LGTM- can you add in the description what the bugs being fixed are and what the cypress tests are testing for?

**Comment by kavithacm** [comment]
> LGTM- can you add in the description what the bugs being fixed are and what the cypress tests are testing for?

yeah okay

File: dashboards-observability/public/components/explorer/explorer.tsx [hunk 1]
```diff
@@ -1051,20 +1090,44 @@ export const Explorer = ({
...
     setIsLiveTailPopoverOpen(false);
+    setLiveTimestamp(dateMath.parse(endTime)?.utc().format(DATE_PICKER_FORMAT));
+    setLiveHits(0);
     await sleep(2000);
     const curLiveTailname = liveTailNameRef.current;
     while (isLiveTailOnRef.current === true && curLiveTailname === liveTailNameRef.current) {
       handleLiveTailSearch(startTime, endTime);
-      if (liveTailTabIdRef.current !== curSelectedTabId.current) {
-        setIsLiveTailOn(!isLiveTailOnRef.current);
+      if ((liveTailTabIdRef.current !== curSelectedTabId.current)) {
```
Comment (eugenesk24) [inline dashboards-observability/public/components/explorer/explorer.tsx:1099]: I think these parentheses are unnecessary

**Review by eugenesk24 — APPROVED** [review]
Good good~ 🚢

File: dashboards-observability/public/components/explorer/explorer.tsx (hunk 1 above)
Comment (kavithacm) [reply dashboards-observability/public/components/explorer/explorer.tsx:1099]: yeah removed one condition from there and forgot to remove the extra

**Pull request merged**